generic/htx_test.py

```diff
@@ -56,8 +56,8 @@ def setUp(self):
             packages.extend(['gcc-c++', 'ncurses-devel',
                              'dapl-devel', 'libcxl-devel'])
         elif detected_distro.name == "Ubuntu":
-            packages.extend(['libncurses5', 'g++', 'libdapl-dev', 'ncurses-dev',
-                             'libncurses-dev', 'libcxl-dev'])
+            packages.extend(['libncurses5', 'g++', 'libdapl-dev',
+                             'ncurses-dev', 'libncurses-dev', 'libcxl-dev'])
```

Comment:
pep8 and pylint fixes generally go as a separate commit, as per previous PRs. Consider that

Reply:
@narasimhan-v sure make sense